src/Files.Uwp/Views/MainPage.xaml.cs

```diff
@@ -177,6 +177,10 @@ public void TabItemContent_ContentChanged(object sender, TabItemArguments e)
             if (SidebarAdaptiveViewModel.PaneHolder != null)
             {
                 var paneArgs = e.NavigationArg as PaneNavigationArguments;
+                if (SidebarAdaptiveViewModel.SidebarSelectedItem is FileTagItem tag)
```

Comment:
Should this be in `SidebarViewModel`?

Reply:
I tried putting this 'if' inside the ViewModel, but the problem wasn't there. Everything is fine until this method gets called at the end of all other invocations.
When you click on a FileTag, paneArgs still has "Home" as the directory searched, and the one to be highlighted. 
This happens because, as you can see in Issue's screenshot, clicking on a Tag results in a search in the "Home".
By checking if the selectedItem is a FileTagItem and, if true, updating the path, I fixed the problem

Reply:
I think the preferred solution would be to change the pane args to match the tag.

Reply:
Ok, I'll try to fix it

Reply:
I should have fixed it
Tell me if that's ok